apps/root/src/services/web3Service.ts

```diff
@@ -424,10 +414,9 @@ export default class Web3Service {
 
     this.setAccount('');
```

Comment:
I understand that handling the account here is useful for re-evaluating components, but perhaps removing `getAccount` method could prevent future confusion regarding where to obtain account data

Reply:
Sure!